content/en/observability_pipelines/reference/processing_language/_index.md

```diff
@@ -1,33 +1,34 @@
 ---
-title: Processing Language
+title: Datadog Processing Language
 ---
 
-Datadog Processing Language (DPL) is an expression-oriented, domain specific language designed for transforming observability data (logs and metrics). It features a simple syntax and [built-in functions][1] tailored to observability use cases.
+The Datadog Processing Language (DPL) is an expression-oriented, domain-specific language designed for transforming data (logs and metrics (beta)) sent to Observability Pipelines. It features a simple syntax and [built-in functions][1].
```

Comment:
```suggestion
The Datadog Processing Language (DPL) is an expression-oriented, domain-specific language designed for transforming data (logs and metrics in beta) sent to Observability Pipelines. It features a simple syntax and [built-in functions][1].
```

Reply:
Going to leave this as is so that it's consistent with the other docs pages where `metrics (beta)` is used.